server/app/controllers/dev/seeding/CategoryTranslationFileParser.java

```diff
@@ -40,6 +42,10 @@ public final class CategoryTranslationFileParser {
   private Map<String, String> utilitiesMap = new HashMap<>();
   private static final Logger logger = LoggerFactory.getLogger(CategoryTranslationFileParser.class);
 
+  public CategoryTranslationFileParser(Environment environment) {
+    this.environment = environment;
```

Comment:
```suggestion
    this.environment = com.google.common.base.Preconditions.checkNotNull(environment);
```

You can import the namespace, just including it to make it easier have.

Reply:
Done